src/dns/server.rs

```diff
@@ -1263,8 +1275,9 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn system_forwarder() {
+    async fn forward_to_server() {
```

Comment:
nit: wdyt about `forward_to_local_server`, if the fact that it is local is truly significant as suggested in the below comment

Reply:
Its not significant to the test, just to general testing philosophy (make tests hermetic)